src/main/java/app/bpartners/geojobs/endpoint/event/model/AutoTaskStatisticRecomputingSubmitted.java

```diff
@@ -2,36 +2,21 @@
 
 import app.bpartners.geojobs.endpoint.event.EventStack;
 import java.time.Duration;
-import lombok.*;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
 
-@NoArgsConstructor
 @AllArgsConstructor
-@Builder
 @Getter
 @EqualsAndHashCode(callSuper = false)
 @ToString
-public class AutoTaskStatisticRecomputingSubmitted extends PojaEvent {
+public class AutoTaskStatisticRecomputingSubmitted extends StatisticRecomputingSubmitted {
```

Comment:
Why not just upadte AutoTaskStatistic without creating StatisticRecomputingSubmitted ?? Because, here the difference is just inside the AutoTaskStatistic you give the default value, and you do not override the default constructor